samples/peripheral/radio_test/README.rst

```diff
@@ -56,6 +56,17 @@ The sample also requires one of the following testing devices:
    The radio test can be also performed using a spectrum analyzer.
```

Comment:
Prefer active voice: You can perform the radio test also using a spectrum analyzer.

Reply:
Ok. Done